workflows/cloudify_system_workflows/deployment_update/workflow.py

```diff
@@ -247,6 +247,39 @@ def _format_instance_relationships(node_instance):
     ]
 
 
+def reorder_node_instance_relationships(*, update_id):
+    client = get_rest_client()
+    dep_up = client.deployment_updates.get(update_id)
+    node_reorders = defaultdict(dict)
+    for step in dep_up.steps:
+        # find steps that modify relationships, with entity_id like
+        # nodes:node_id:relationships:[1]:[0] - that means relationship
+        # at index=1 is to be moved to index=0
+        if step['action'] != 'modify' or step['entity_type'] != 'relationship':
+            continue
+        parts = step['entity_id'].split(':')
+        if len(parts) != 5:
+            continue
+        _, node_id, _, from_ix, to_ix = parts
```

Comment:
Maybe we should verify the first and the third element are `nodes` and `relationships`?  Just to be on the safe side (sometime, somewhere)